client/state/login/reducer.js

```diff
@@ -51,7 +74,17 @@ const updateTwoStepNonce = ( state, { twoStepNonce, nonceType } ) => Object.assi
 
 export const twoFactorAuth = createReducer( null, {
 	[ LOGIN_REQUEST ]: () => null,
-	[ LOGIN_REQUEST_SUCCESS ]: ( state, { data, rememberMe } ) => data ? { ...data, remember_me: rememberMe } : null,
+	[ LOGIN_REQUEST_SUCCESS ]: ( state, { data } ) => {
+		if ( data ) {
+			const rest = omit( data, 'redirect_to' );
```

Comment:
Taking into account all comments we discussed in this PR and concerns related to 2FA, redirect param, the way we validate if 2FA is required, we probably should update the response we send from the server. If we would keep `redirect_to` and `two_factor_auth ` separate to make it would make integration with Redux state easier.